CMP1903_A1_2324/Testing.cs

```diff
@@ -14,7 +16,17 @@ internal class Testing
          * Create a Die object and call its method.
          * Use debug.assert() to make the comparisons and tests.
          */
+        public void testGame()
+        {
+            Game game = new Game(); //creates a new game object
+            int test = game.Play(); //calls the play method from the game class
+
+            Debug.Assert(test == (game.die1.diceValue + game.die2.diceValue + game.die3.diceValue),
+                 "dice total is incorrect");
 
-        //Method
+            Die dice = new Die();
+            int testDie = dice.Roll(); //creates a die object to be tested
+            Debug.Assert(testDie >= 1 && testDie <= 6, "die is incorrect"); //checks whether an individual dice roll is within range
+        }
     }
 }
```

Comment:
meets the requirement for the testing class, uses appropriate outputs for an invalid value and accurately tests the boundaries. good show of error handling.

Reply:
only improvements could be a few more comments, especially in the die class to make the code simpler to read through and understand.